src/api/api.js

```diff
@@ -33,56 +33,40 @@ async function createEvent(event) {
   return res;
 }
 
-const stripePromise = loadStripe(process.env.VUE_APP_STRIPE_PUBLISHABLE_KEY);
+const stripeApp = loadStripe(process.env.VUE_APP_STRIPE_PUBLISHABLE_KEY);
 
-async function handleClickCheckout(cartEvents, userLevel) {
-  if (userLevel === 'ADMIN' || userLevel === 'PF') {
-    const body = {
-      lineItems: cartEvents.map(event => ({
-        id: event.id,
-        name: event.title,
-        description: event.details.description,
-        amount: 5,
-        currency: 'usd',
-        quantity: 1,
-      })),
-      successUrl: `${process.env.VUE_APP_API_DOMAIN}/my-events/?session_id={CHECKOUT_SESSION_ID}`,
```

Comment:
What is the difference between these two?

Reply:
one immediately persists for PF and Admin, one creates a stripe checkout session